src/screens/GroupInfo/hooks/useGroupInfo.js

```diff
@@ -245,7 +239,7 @@ const useGroupInfo = () => {
         `${profile.myProfile.username} removed you from this group`,
         selectedUser.user_id
       );
-      setNewParticipan(result.members);
+      setNewParticipant(result.members);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `setNewParticipant` is now being used instead of `setNewParticipan`. This is a good change as it corrects the typo in the previous function name.

```diff
- setNewParticipan(result.members);
+ setNewParticipant(result.members);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->